runners/flink/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/DoFnOperator.java

```diff
@@ -295,14 +295,10 @@ public void initializeState(StateInitializationContext context) throws Exception
 
     sideInputReader = NullSideInputReader.of(sideInputs);
 
-    // maybe init by initializeState
```

Comment:
Why'd you remove this check?

Reply:
btw, your comment was here twice and I deleted the duplicate :)